test/integration/controllers/v1/ApiController.test.coffee

```diff
@@ -0,0 +1,62 @@
+###*
+# ApiController.test
+###
+
+request = require('supertest')
+should = require('should')
+
+###*
+# Test API "ping" endpoint.
+###
+
+describe 'Requests to the root (/api) path', ->
+  it 'GET: Returns a 200 status code', (done) ->
+    request(sails.hooks.http.app)
+      .get('/api')
```

Comment:
I find the chained items on their own line easier to read. I get lost or miss the details of a chain if it has more than a few items. Am I required to put them all on one line as it appears to compile OK?